templates/prometheus.go

```diff
@@ -88,86 +54,5 @@ var PrometheusTemplate = promv1.Prometheus{
 		},
 		Resources:   utils.GetResourceRequirements("prometheus"),
 		ListenLocal: true,
-		Containers: []corev1.Container{{
```

Comment:
Why remove kube-rbac-proxy

Reply:
Because there is no reason to put the Prometheus behind HTTP
We don't need metrics federation logic anymore

Reply:
ACK